implementationDependencies.json

```diff
@@ -105,6 +105,11 @@
       "jar": "https://repo1.maven.org/maven2/com/eatthepath/java-otp/0.4.0/java-otp-0.4.0.jar",
       "name": "Java OTP 0.4.0",
       "src": "https://repo1.maven.org/maven2/com/eatthepath/java-otp/0.4.0/java-otp-0.4.0-sources.jar"
+    },
+    {
+      "jar": "https://repo1.maven.org/maven2/commons-codec/commons-codec/1.15/commons-codec-1.15.jar",
+      "name": "Commons Codec 1.15",
+      "src": "https://repo1.maven.org/maven2/commons-codec/commons-codec/1.15/commons-codec-1.15-sources.jar"
```

Comment:
adding the jar is not always done this way - we need to see the list of built jars and also add those in here, cause this jar may also pull in other jars

Reply:
It's not pulling anything else. `core-4.4.0.jar` is the only difference between `.jar` files present in `core` dir.